boards/arm/efm32gg_stk3701a/doc/index.rst

```diff
@@ -0,0 +1,203 @@
+| PH11  | GPIO        | LED0 green                          |
++-------+-------------+-------------------------------------+
+| PH12  | GPIO        | LED0 blue                           |
++-------+-------------+-------------------------------------+
+| PH13  | GPIO        | LED1 red                            |
++-------+-------------+-------------------------------------+
+| PH14  | GPIO        | LED1 green                          |
++-------+-------------+-------------------------------------+
+| PH15  | GPIO        | LED1 blue                           |
++-------+-------------+-------------------------------------+
+| PC8   | GPIO        | Push Button PB0                     |
++-------+-------------+-------------------------------------+
+| PC9   | GPIO        | Push Button PB1                     |
++-------+-------------+-------------------------------------+
+| PE1   | GPIO        | Board Controller Enable             |
+|       |             | EFM_BC_EN                           |
++-------+-------------+-------------------------------------+
+| PH4   | UART_TX     | UART TX Console VCOM_TX US0_TX #4   |
++-------+-------------+-------------------------------------+
+| PH5   | UART_RX     | UART RX Console VCOM_RX US0_RX #4   |
++-------+-------------+-------------------------------------+
+| PI4   | I2C_SDA     | SENSOR_I2C_SDA I2C2_SDA #7          |
++-------+-------------+-------------------------------------+
+| PI5   | I2C_SCL     | SENSOR_I2C_SCL I2C2_SCL #7          |
++-------+-------------+-------------------------------------+
+
+
+System Clock
+============
+
+The EFM32GG11 SoC is configured to use the 50 MHz external oscillator on the
+board.
+
+Serial Port
+===========
+
+The EFM32GG11 SoC has six USARTs, two UARTs and two Low Energy UARTs (LEUART).
+
+Programming and Debugging
+*************************
+
+.. note::
+   Before using the kit the first time, you should update the J-Link firmware
+   from `J-Link-Downloads`_
+
+Flashing
+========
+
+The EFM32GG-STK3701A includes an `J-Link`_ serial and debug adaptor built into the
+board. The adaptor provides:
+
+- A USB connection to the host computer, which exposes a mass storage device and a
+  USB serial port.
+- A serial flash device, which implements the USB flash disk file storage.
+- A physical UART connection which is relayed over interface USB serial port.
+
+Flashing an application to EFM32GG-STK3701A
+-------------------------------------------
+
+The sample application :ref:`hello_world` is used for this example.
+Build the Zephyr kernel and application:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: efm32gg_stk3701a
+   :goals: build
+
+Connect the EFM32GG-STK3701A to your host computer using the USB port and you
+should see a USB connection which exposes a mass storage device(STK3701A).
+Copy the generated zephyr.bin to the STK3701A drive.
+
+Use a USB-to-UART converter such as an FT232/CP2102 to connect to the UART on the
```

Comment:
Do you need to clarify which pins on the header to connect to?

Reply:
This is definately a case of 'copy/paste'. The console is assigned to USART4, which is exposed through the USB connector along with the mass storage device. So corrected that and removed any mention of the USB-to-UART converter.